### PR TITLE
Building the class: Product

### PR DESCRIPTION
**Pull request by kpaxiotis**
I built the basic functions of the Product class.
I defined the price of the product as a float both as an instance variable and a class variable (override?).
Is it ok?

### CODE REVIEW

**Comment by ngiachou** [comment]
Nice! Very good first try! I will review it tommorow and I will let you know.

You should also reference the issue that this PR solves. For example, **solves** #3

File: modules/Product.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Product:
+    def __init__(self, name, price:float):
+        self.name = name
+        self.price = float(price)
+
```
Comment (ngiachou) [inline modules/Product.py]: Everything inside `def main()` should be under the `if __name__ == "__main__"` and not on a different method. You can also leave it as is (without having it inside a comment).

**Review by ngiachou — COMMENTED** [review]
Just testing this review functionality lol 😛

**Comment by ngiachou** [comment]
Good work @kpaxiotis but next time it is better to start a new branch for the staff you implement. Thanks!

**Comment by kpaxiotis** [comment]
Should I create a new brunch and build the class there or is it ok for now?

**Comment by ngiachou** [comment]
@kpaxiotis nicely done. Let's leave it on this branch, because with a new one a new Pull Request is needed to and it will get too complicated.

**Pull request merged**